stdlib/REPL/src/latex_symbols.jl

```diff
@@ -956,7 +960,6 @@ const latex_symbols = Dict(
     "\\ocommatopright" => "̕",  # combining comma above right
     "\\droang" => "̚",  # left angle above (non-spacing)
     "\\wideutilde" => "̰",  # under tilde accent (multiple characters and non-spacing)
-    "\\underbar" => "̱",  # combining macron below
```

Comment:
This and the other `\underbar` are actually different characters. Is there a way to keep both?

Reply:
Yes, we could the "combining macron below" (the shorter underbar) a different name.

But no one seems to have needed the shorter one up to now (since it was overwritten by the other `\underbar` definition), and I couldn't come up with a great name, so to stay on the conservative side I just left it out.